src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+  max-width: 1199px;
```

Comment:
don't do it (max-width)

Reply:
but then [this (https://prnt.sc/19q26af) happened
and I have to move it to .container to pass [this](https://prnt.sc/19q2nn0) test